app/policies/person_policy.rb

```diff
@@ -1,2 +1,24 @@
 class PersonPolicy < ApplicationPolicy
+
+  # Make records assocaited with a user record read only except for admins or self.
+  def update?
+    user_is_record? || @record.has_system_access ? division_admin : division_member_or_admin
+  end
+
+  def update_access?
+    division_admin
+  end
+
+  def update_password?
+    user_is_record? || division_admin
+  end
+
+  private
+
+  def user_is_record?
+    @record.user && @user && @user.id != nil && @record.user.id == @user.id
```

Comment:
I was wanting to guarantee to never match when `@record.user` was nil, as I've been burned in the past in a similar context.  My tendency is to code a bit defensively, but I agree that it's probably not strictly needed here.  I'll simplify.